src/backend/commands/tablecmds.c

```diff
@@ -3948,9 +3948,9 @@ RenameRelation(RenameStmt *stmt)
 	 * escalation.  However, because ALTER INDEX can be used with any relation
 	 * type, we mustn't believe without verification.
 	 */
+	LOCKMODE	lockmode;
```

Comment:
Why move this declaration out?

Reply:
To reuse the same lockmode for index statement at line 4000.